InterfaceParser/model/enum.py

```diff
@@ -0,0 +1,40 @@
+"""Enumeration.
+
+"""
+from collections import OrderedDict
+
+from model.interface_item_base import InterfaceItemBase
+
+
+class Enum(InterfaceItemBase):
+    """Enumeration.
+
+    :param name: item name
+    :param description: list of string description elements
+    :param design_description: list of string design description elements
+    :param issues: list of issues
+    :param todos: list of string todo elements
+    :param platform: optional platform (string or None)
+    :param scope: optional scope: internal, partner or none (none by defaul, means public)
+    :param since:
```

Comment:
Could you please add descriptions for since, until, deprecated, removed, and history?

Let me know if you would like some help with those items.

Reply:
Same comment applies to all other files that include these "versioning" parameters without descriptions

Reply:
@JackLivio I can turn my imagination to the maximum and invent a supposition of what each parameter mean based on it name, but I'll really appreciate if you will provide description for each parameter. Then I'll be able to add descriptions provided by you into code.

Reply:
Here is the proposal that introduced those terms to the RPC spec: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0089-mobile-api-versioning.md. This should help serve as a guide for the descriptions.

Reply:
@o-mishch ^

Reply:
appropriate descriptions added added